.github/workflows/ruff_linter.yml

```diff
@@ -33,43 +33,43 @@ jobs:
         PR_URL=${{ github.event.inputs.pr_url }}
         PR_NUMBER=$(echo $PR_URL | grep -oE '[0-9]+$')
         echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
-        
+
     - uses: actions/checkout@v3
       if: github.event_name == 'workflow_dispatch'
       with:
         fetch-depth: 0
         token: ${{ secrets.GITHUB_TOKEN }}
-    
+
     - name: Checkout PR branch
       if: github.event_name == 'workflow_dispatch'
       run: |
         gh pr checkout ${{ env.PR_NUMBER }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        
+
     - uses: actions/checkout@v3
       if: github.event_name != 'workflow_dispatch'
       with:
         fetch-depth: 0
-        
+
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
-        
+
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         pip install ruff==0.6.8
-        
+
     - name: Regular lint check
       if: github.event_name != 'workflow_dispatch'
       run: |
         ruff check .
         # --isolated is used to skip the allowlist at all so this applies to all files
         # please be careful when using this large changes means everyone needs to rebase
         ruff check --isolated --select F821,F823,W191
-        ruff check --select F,I
+        ruff check
```

Comment:
`ruff check` is different from `ruff check .` above?

Reply:
ohh good catch